DataFormats/JetReco/src/classes_def_4.xml

```diff
@@ -138,4 +138,20 @@
    <class name="edm::PtrVector<reco::FFTCaloJet>" />
    <class name="edm::Association<reco::FFTCaloJetCollection>" />
    <class name="edm::Wrapper<edm::Association<reco::FFTCaloJetCollection> >" />
+
+   <class name="reco::HTTTopJetProperties" ClassVersion="4">
+     <version ClassVersion="4" checksum="3737009156"/>
+   </class>
+   <class name="std::pair<edm::RefToBase<reco::Jet>, reco::HTTTopJetProperties>"/>
+  
+   <class name="reco::HTTTopJetTagInfo" ClassVersion="5">
+     <version ClassVersion="5" checksum="3013029993"/>
```

Comment:
I had to increment when we changed the HTTTopJetProperties (consistent case of variables)

Should I reset back to 3?